example_data/sample_bundle.json

```diff
@@ -20,12 +30,17 @@
             "text": "NHS UK Healthcare Provider"
           }
         ],
-        "name": "Great Ormond Street Hospital for Children NHS Foundation Trust",
+        "name": "North London Lab 1",
+        "telecom": [
+          {
+            "system": "phone",
+            "value": "2078298870"
+          }
+        ],
```

Comment:
is the phone number useful? I'm concerned about adding in extra information that won't be useful for any data sharing activities but just makes life harder for getting the data in

Reply:
Sure. will be removed then.